### PR TITLE
First version - no tests

### PR DESCRIPTION
**Pull request by thehh1974**
First version builds a docker image with the adapter and octactl, including the necessary credentials.
No tests yet.

### CODE REVIEW

File: .octactl.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+api_host: us-112.octarinesec.com
+api_port: 443
+config_version: 0.11.2
+docker:
+  email: kubecon@octarinesec.com
+  password: V2VsY29tZVRvT2N0YXJpbmU=
+  username: octarinekubecon
```
Comment (leecalcote) [inline .octactl.yaml]: Recognizing that a public use control plane is underway, are you good with these credentials being public in the meantime?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+proto:	
+	protoc -I meshes/ meshes/meshops.proto --go_out=plugins=grpc:./meshes/
+
+docker:
+	docker build -t layer5/meshery-octarine .
+
+docker-run:
+	(docker rm -f meshery-octarine) || true
+	docker run --name meshery-octarine -d \
+	-p 10000:10000 \
```
Comment (leecalcote) [inline Makefile]: @girishranganathan does @thehh1974 need to make use of port 10004 instead?

(istio - 10000)
(linkerd - 10001)
(consul - 10003)
(octarine -10004)
(nsm - 10005)

File: octarine/install-octarine.go [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+const (
+	nsMgrUsername      = "meshery"
+	nsMgrPassword      = "meshery-password"
+	octarineDeployment = "meshery:playground"
```
Comment (leecalcote) [inline octarine/install-octarine.go]: @thehh1974 quick check - are we good with having these credentials public?

File: .octactl.yaml (hunk 1 above)
Comment (girishranganathan) [reply .octactl.yaml]: yeah it will be good to move these to either run time params or environment variables

File: Makefile (hunk 2 above)
Comment (girishranganathan) [reply Makefile]: Yes, definitely a nice to have to use a different port but not a mandate.

File: octarine/install-octarine.go (hunk 3 above)
Comment (girishranganathan) [reply octarine/install-octarine.go]: Better to have it as run time params or environment variables which are injected dynamically.

**Comment by leecalcote** [comment]
Removed sensitive text.  Proceeding with merge for alpha version.

**Pull request merged**